src/bundle/Command/MigrateNamespacesCommand.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\FieldTypeRichText\Command;
+
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\FieldTypeRichText\Persistence\Legacy\ContentModelGateway as Gateway;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class MigrateNamespacesCommand extends AbstractMultiProcessComand
+{
+    private Gateway $gateway;
+
+    private ?int $cursorStart;
+
+    private ?int $cursorStop;
+
+    private ?int $objectCount;
+
+    public function __construct(
+        PermissionResolver $permissionResolver,
+        UserService $userService,
+        Gateway $gateway
+    ) {
+        $this->objectCount = null;
+        parent::__construct('ibexa:migrate:richtext-namespaces', $permissionResolver, $userService);
+        $this->gateway = $gateway;
+    }
+
+    public function configure(): void
+    {
+        parent::configure();
+
+        $this->addOption(
+            'cursor-start',
+            null,
+            InputOption::VALUE_REQUIRED,
+            'Internal option - only used for subprocesses',
+        )
+            ->addOption(
+                'cursor-stop',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Internal option - only used for subprocesses',
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->cursorStart = $input->getOption('cursor-start') !== null ? (int) $input->getOption('cursor-start') : null;
+        $this->cursorStop = $input->getOption('cursor-stop') !== null ? (int) $input->getOption('cursor-stop') : null;
+
+        // Check that both --cursor-start and cursor-start are set, or neither
+        if (($this->cursorStart === null) xor ($this->cursorStop === null)) {
+            throw new RuntimeException('The options --cursor-start and -cursor-stop are only for internal use !');
+        }
+
+        parent::execute($input, $output);
+
+        return self::SUCCESS;
+    }
+
+    protected function getObjectCount(): int
+    {
+        if ($this->objectCount === null) {
+            $this->output->writeln("Fetching number of objects to process. This might take several minutes if you are have many records in ezcontentobject_attribute table.");
```

Comment:
```suggestion
            $this->output->writeln("Fetching number of objects to process. This might take several minutes if you have many records in ezcontentobject_attribute table.");
```

Reply:
Thanx. Fixed.